### PR TITLE
Ensure that aesm and launch are installed in non dcap

### PR DESCRIPTION
**Pull request by yakman2020**
This is not yet covered in the document and is non-obvious.  We have an example of the problem in #2624.

### CODE REVIEW

**Comment by jhand2** [comment]
Why can't this just be done in the ansible script? Our setup docs don't specifically call out any other packages that should be installed, they just say to use ansible.

File: docs/GettingStartedDocs/Contributors/SGX1GettingStarted.md [hunk 1]
```diff
@@ -7,6 +7,13 @@
 - Red Hat Enterprise Linux 8 64-bits
 - SGX1 capable system. Most likely this will be an Intel SkyLake or Intel KabyLake system
 
+## Prepare the machine for sgx 
+
+Ensure that the aesm service and lanuch enclave are properly installed.
```
Comment (dthaler) [inline docs/GettingStartedDocs/Contributors/SGX1GettingStarted.md]: typo: "lanuch"

File: docs/GettingStartedDocs/Contributors/SGX1GettingStarted.md [hunk 2]
```diff
@@ -7,6 +7,13 @@
 - Red Hat Enterprise Linux 8 64-bits
 - SGX1 capable system. Most likely this will be an Intel SkyLake or Intel KabyLake system
 
+## Prepare the machine for sgx
```
Comment (dthaler) [inline docs/GettingStartedDocs/Contributors/SGX1GettingStarted.md]: nit: capitalize SGX

**Comment by BRMcLaren** [comment]
bors try

**Comment by BRMcLaren** [comment]
I am wondering if we need to add a test case for this to the nightly builds. I agree it is of value, but I am conflicted about merging anything if we don't have a test for it.

Id we were to add the ansible scripts like @jhand2 suggested and we use the principles of having multilevel testing I think that would be really strong.

**Comment by jhand2** [comment]
> I am wondering if we need to add a test case for this to the nightly builds. I agree it is of value, but I am conflicted about merging anything if we don't have a test for it.
> 
> If we were to add the ansible scripts like @jhand2 suggested and we use the principles of having multilevel testing I think that would be really strong.

Doesn't this PR just change documentation?

**Comment by BRMcLaren** [comment]
@jhand2 Maybe I misunderstand. Is this documentation that reflects the manual set up of [this](https://github.com/openenclave/openenclave/pull/2604) pull request?

**Comment by jhand2** [comment]
> @jhand2 Maybe I misunderstand. Is this documentation that reflects the manual set up of [this](https://github.com/openenclave/openenclave/pull/2604) pull request?

Yes I believe that is correct. My point is I don't think it's necessary and is potentially confusing since we don't document this for any other packages

**Comment by kuanggang-git** [comment]
Can one of the admins verify this patch?

**Pull request closed without merging**